src/map/lua/lua_baseentity.cpp

```diff
@@ -8697,6 +8679,16 @@ inline int32 CLuaBaseEntity::hasSpellList(lua_State* L)
     return 1;
 }
 
+inline int32 CLuaBaseEntity::hasPreventActionEffect(lua_State* L)
+{
+    DSP_DEBUG_BREAK_IF(m_PBaseEntity == nullptr);
+    DSP_DEBUG_BREAK_IF(m_PBaseEntity->objtype != TYPE_MOB && m_PBaseEntity->objtype != TYPE_PET && 
+        m_PBaseEntity->objtype != TYPE_PC);
+
+    lua_pushboolean(L, ((CMobEntity*)m_PBaseEntity)->StatusEffectContainer->HasPreventActionEffect());
```

Comment:
This should probably be changed in other places as well then, since I copied this code from....hasSpellList() I think.